internal/c/libqb.cpp

```diff
@@ -27544,28 +27544,34 @@ void GLUT_KEYBOARDUP_FUNC(unsigned char key, int x, int y) { GLUT_key_ascii(key,
 
 void GLUT_key_special(int32 key, int32 down) {
 #ifdef QB64_GLUT
+
+#    ifdef QB64_WINDOWS
+    if (!func__hasfocus())

```

Comment:
Previous versions will still report the release even if focus is lost while the key is held. I think we can match that if we move this past the `vk = ` conversion logic and then do `if (!func__hasfocus() && !(keyheld(vk) && !down))`

Reply:
Made changes as suggested.